mlxtend/plotting/plot_sequential_feature_selection.py

```diff
@@ -50,12 +53,17 @@ def plot_sequential_feature_selection(metric_dict,
     http://rasbt.github.io/mlxtend/user_guide/plotting/plot_sequential_feature_selection/
 
     """
-
+    
+    if figsize is None:
+        figsize = (10,10)
```

Comment:
Hm, I think it would be better not to choose a particular figsize if it is `None`, otherwise it will be confusing to users. I suggest 

```python
if figsize is not None:
    fig = plt.subplots(figsize=figsize)
else:
    fig = plt.subplots()
```

Reply:
I agree, it should be like this now